### PR TITLE
Proposal for edits to the revised charter

### PR DESCRIPTION
**Pull request by cportele**

### CODE REVIEW

**Comment by chris-little** [comment]
@cportele stadnards -> standards
Also, I would prefer 
`OGC API Standards specify Web based capabilities which are typically based on polling for collection resource updates (new or changed or deleted features/records, coverages, maps, etc.). Depending on a collection's temporal resolution or frequency of updates, an event-driven / Publish-Subscribe architecture provides a timely, efficient and low latency approach for delivery of data updates.`
to read something like:
`OGC API Standards specify Web based capabilities which are typically based on polling for collection resource updates (new or changed or deleted features/records, coverages, maps, data, etc.). Depending on a collection's temporal resolution or frequency of updates or size, an event-driven / Publish-Subscribe architecture provides a timely, efficient and low latency approach for delivery of data updates or notifications.`

**Review by ghobona — APPROVED** [review]

File: charter/PubSub_SWG_Charter.adoc [hunk 1]
```diff
@@ -113,7 +116,7 @@ This proposed standard is intended to be a major component of the OGC API framew
 
 === What is out of scope?
 
-The scope of the {titletext} is confined to enabling basic publish/subscribe functionality for existing and future OGC APIs. New filtering techniques will not be addressed, but adaptation of existing filtering mechanisms may be included. Advanced concepts such as pubsub security and Complex Event Processing are out of scope. Any other requirement or functionality the SWG decides to be not contributing to core publish/subscribe functionality is also out of scope.
+The scope of the {titletext} is confined to enabling basic publish/subscribe functionality for resources from existing and future OGC API Stadnards. New filtering techniques will not be addressed, but adaptation of existing filtering mechanisms may be included. Advanced concepts such as pubsub security and Complex Event Processing are out of scope. Any other requirement or functionality the SWG decides to be not contributing to core publish/subscribe functionality is also out of scope.
```
Comment (tomkralidis) [inline charter/PubSub_SWG_Charter.adoc:119]: `s/Stadnards/Standards/g`

**Review by tomkralidis — CHANGES_REQUESTED** [review]

File: charter/PubSub_SWG_Charter.adoc (hunk 1 above)
Comment (cportele) [reply charter/PubSub_SWG_Charter.adoc]: Thanks, fixed.

**Comment by cportele** [comment]
@chris-little - Thanks. I have fix the typo and added two of your three changes. I have not added "data", because it is a generic term and not a resource type in the OGC API standards.

**Review by chris-little — COMMENTED** [review]
@cportele Thank you. But `data` is a resource type for OGC API-EDR.

**Pull request merged**